livekit/src/room/participant/local_participant.rs

```
@@ -331,6 +333,56 @@ impl LocalParticipant {
         }
     }
 
+    pub async fn send_chat_message(
+        &self,
+        text: String,
+        destination_identities: Option<Vec<String>>,
+        sender_identity: Option<String>,
+    ) -> RoomResult<ChatMessage> {
+        let chat_message = proto::ChatMessage {
+            id: create_random_uuid(),
+            timestamp: Utc::now().timestamp_millis(),
+            message: text,
+            ..Default::default()
+        };
+
+        let data = proto::DataPacket {
+            value: Some(proto::data_packet::Value::ChatMessage(chat_message.clone())),
+            participant_identity: sender_identity.unwrap_or_default(),
+            destination_identities: destination_identities.unwrap_or_default(),
+            ..Default::default()
+        };
+
+        let _ = self.inner.rtc_engine.publish_data(&data, DataPacketKind::Reliable).await;
```

We shouldn't ignore the error here